Modix.Bot/Modules/QuoteModule.cs

```diff
@@ -63,6 +71,12 @@ public class QuoteModule : ModuleBase
             try
             {
                 message = await GetMessage(messageId, channel);
+
+                var user = Context.User as IGuildUser;
```

Comment:
Same nitpick as above

```suggestion
                var user = (IGuildUser)Context.User;
```